src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -507,10 +519,23 @@ public static T Attach<T>(Stream stream)
         public static T Attach<T>(Stream sendingStream, Stream receivingStream)
             where T : class
         {
+            string typeName = typeof(T).FullName;
+
+            if (JsonRpcEventSource.Instance.IsEnabled())
+            {
+                JsonRpcEventSource.Instance.GenerateProxyUsingStreamsAndStartListeningStart(typeName);
```

Comment:
I'm not a fan of all these events that basically just serve as bounds for a method call. CPU stack events already give this idea. Why do you need extra start and stop events for these methods?

Reply:
Removed these events as per our discussion yesterday.